CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## v0.16.1 (2017-6-19)
+
+### Bug Fixes
+
+* We need to process a markup imported via `meta[rel="import"]` ([testcafe-hammerhead/#1161](https://github.com/DevExpress/testcafe-hammerhead/issues/1161))
+* Wrong passing context to a `MutationObserver` ([testcafe-hammerhead/#1178](https://github.com/DevExpress/testcafe-hammerhead/issues/1178))
+* We don't need to process an `innerHTML` property if element doesn't have it ([testcafe-hammerhead/#1164](https://github.com/DevExpress/testcafe-hammerhead/issues/1164))
```

Comment:
* Markup imported via meta[rel="import"] is now processed. ([testcafe-hammerhead/#1161](https://github.com/DevExpress/testcafe-hammerhead/issues/1161))
* The correct context is passed to `MutationObserver`. ([testcafe-hammerhead/#1178](https://github.com/DevExpress/testcafe-hammerhead/issues/1178))
* The `innerHtml` property is no longer processed for elements that don't have this property.  ([testcafe-hammerhead/#1164](https://github.com/DevExpress/testcafe-hammerhead/issues/1164))